src/recordlinker/linking/link.py

```diff
@@ -0,0 +1,181 @@
+"""
+recordlinker.linking.link
```

Comment:
Its a temporary change so we can have two schemas at the same time (linkage == old schema code, linking == new schema code).  Long term, it will just be `recordlinker.linkage`.